lib/curl_setup_once.h

```diff
@@ -268,6 +268,7 @@ struct timeval {
 #endif
 
 #define TOLOWER(x)  (tolower((int)  ((unsigned char)x)))
+#define TOUPPER(x)  (toupper((int)  ((unsigned char)x)))
```

Comment:
Why do need this code?

Reply:
good question, I've take inspiration from what is done in `urlapi.c`, where TOLOWER is use there.
TBH I don't know what's the advantage over using `toupper`, I've just assume it was there for a reason, maybe it was wrong, @bagder ?

Reply:
You want to use `Curl_raw_toupper()` and not `toupper()` to avoid it being affected by locales.